demo/demo_primary.py

```diff
@@ -559,37 +559,27 @@ def get_image_for_ecu(ecu_serial):
 
 def get_metadata_for_ecu(ecu_serial, force_partial_verification=False):
   """
-  Send a zip archive of the most recent consistent set of the Primary's client
-  metadata directory, containing the current, consistent metadata from all
-  repositories used.
+  Provides the (updated) metadata a Secondary will need to validate updates.
```

Comment:
Is it worth mentioning what *updated* means?

Reply:
Hmm. Replacing with current in 78e8d2c.

Adding these all in #148.